lib/record_store/record/txt.rb

```diff
@@ -2,28 +2,37 @@ module RecordStore
   class Record::TXT < Record
     attr_accessor :txtdata
 
-    validates :txtdata, presence: true, length: { maximum: 255 }
-    validate :escaped_semicolons
+    validates :txtdata, presence: true, length: { maximum: 4096 }
```

Comment:
Is this an arbitrary limit? I took a cursory look at [RFC 4408](https://datatracker.ietf.org/doc/rfc4408/?include_text=1) and I didn't see any obvious length limit.

Reply:
From your link:

>   SPF or TXT records containing multiple strings are useful in
   constructing records that would exceed the 255-byte maximum length of
   a string within a single TXT or SPF RR record

Reply:
Yes, I was referring to the 4096 byte limit. :)

Reply:
Huh my bad. Should have read what line you commented on!

Reply:
I can't find the reference any more, but I had read that there was a general limit of 4096 bytes on any DNS record.

..but then, I see that [RDLENGTH](https://www.rfc-editor.org/rfc/rfc1035.html#section-3.2.1) is a 16-bit integer..so really, the theoretical max size of a record is 65535 bytes.

I'll look this up again, and adjust as needed.

Reply:
I'm OK with a 4096 byte limit. In practice, this won't be common and we can always bump the limit if needed.